From 2f89c292fc214555bcfb59a44dc7b10cc19857ac Mon Sep 17 00:00:00 2001
From: Dan Printzell <xwildn00bx@gmail.com>
Date: Fri, 17 Jan 2020 01:49:05 +0100
Subject: [PATCH 3/4] Force PowerNex to be the target

Signed-off-by: Dan Printzell <xwildn00bx@gmail.com>
---
 src/dmd/globals.d | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/dmd/globals.d b/src/dmd/globals.d
index ceeb1b863..5832bd909 100644
--- a/src/dmd/globals.d
+++ b/src/dmd/globals.d
@@ -40,7 +40,8 @@ enum TargetOS : ubyte
 
 template xversion(string s)
 {
-    enum xversion = mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
+    // enum xversion = s == mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
+    enum xversion = s == "PowerNex";
 }
 
 enum TARGET : bool
-- 
2.30.2

