From cb7d29ecccfe500480241614ce07b0b099eaf375 Mon Sep 17 00:00:00 2001
From: Dan Printzell <xwildn00bx@gmail.com>
Date: Fri, 17 Jan 2020 01:49:05 +0100
Subject: [PATCH 3/4] Force PowerNex to be the target

Signed-off-by: Dan Printzell <xwildn00bx@gmail.com>
---
 src/dmd/globals.d | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/dmd/globals.d b/src/dmd/globals.d
index b340de06b..3bf70a134 100644
--- a/src/dmd/globals.d
+++ b/src/dmd/globals.d
@@ -19,7 +19,8 @@ import dmd.identifier;
 
 template xversion(string s)
 {
-    enum xversion = mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
+    // enum xversion = s == mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
+    enum xversion = s == "PowerNex";
 }
 
 enum TARGET : bool
-- 
2.30.1

