From f6e9407181b537a4097750b3971e8d7ae3431148 Mon Sep 17 00:00:00 2001
From: Dan Printzell <xwildn00bx@gmail.com>
Date: Fri, 25 Aug 2017 02:15:44 +0200
Subject: [PATCH 3/4] Force PowerNex to be the target

Signed-off-by: Dan Printzell <xwildn00bx@gmail.com>
---
 src/ddmd/globals.d | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ddmd/globals.d b/src/ddmd/globals.d
index 88aafcc8f..f05b6df83 100644
--- a/src/ddmd/globals.d
+++ b/src/ddmd/globals.d
@@ -20,7 +20,7 @@ import ddmd.root.outbuffer;
 
 template xversion(string s)
 {
-    enum xversion = mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
+    enum xversion = (s == "PowerNex"); //mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
 }
 
 enum IN_GCC     = xversion!`IN_GCC`;
-- 
2.14.1

