From a5432c41af6961b4adda04da98a68d5617bfffb8 Mon Sep 17 00:00:00 2001
From: Dan Printzell <xwildn00bx@gmail.com>
Date: Fri, 25 Aug 2017 02:15:44 +0200
Subject: [PATCH 3/3] Force PowerNex to be the target

Signed-off-by: Dan Printzell <xwildn00bx@gmail.com>
---
 src/ddmd/globals.d | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ddmd/globals.d b/src/ddmd/globals.d
index 922c5baf3..739d76f90 100644
--- a/src/ddmd/globals.d
+++ b/src/ddmd/globals.d
@@ -18,7 +18,7 @@ import ddmd.root.outbuffer;
 
 template xversion(string s)
 {
-    enum xversion = mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
+    enum xversion = (s == "PowerNex"); //mixin(`{ version (` ~ s ~ `) return true; else return false; }`)();
 }
 
 enum IN_GCC     = xversion!`IN_GCC`;
-- 
2.14.1

